src/Fixer/Alias/ArrayPushFixer.php

```diff
@@ -197,7 +197,9 @@ private function getSecondArgumentEnd(Tokens $tokens, int $index, int $endIndex)
             return null;
         }
 
-        $index = $tokens->getNextMeaningfulToken($index);
+        if (!$tokens[$index]->isGivenKind(CT::T_ARRAY_SQUARE_BRACE_OPEN)) {
```

Comment:
>$index = $tokens->getNextMeaningfulToken($index);

removing this seems good to me :+1: 
however I think this `if` block is not needed either

Reply:
@SpacePossum  : I checked your suggestion. Everything looks right.

Reply:
thanks so much for the PR :+1:  RTM from me : )